components/dot_menu/index.ts

```diff
@@ -103,7 +105,23 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
 
     const apps = appsEnabled(state);
     const showBindings = apps && !systemMessage && !isCombinedUserActivityPost(post.id);
-    const appBindings = showBindings ? getPostMenuBindings(state) : undefined;
+    let appBindings = null;
```

Comment:
here as well, I feel like we can handle all this logic in a new action and just pass it as props to the component, which is the intended redux approach.